evap/grades/templates/grades_base.html

```diff
@@ -7,5 +7,9 @@
 
 {% block subtitle %}
     {{ block.super }}
-    <li><a href="{% url "grades:index" %}">{% trans "Grade publishing" %}</a></li>
+    {% if is_grade_publisher %}
+    	<li><a href="{% url "grades:index" %}">{% trans "Grade publishing" %}</a></li>
+    {% else %}
+		<li>{% trans "Grade publishing" %}</li>
```

Comment:
tab spotted here. also, why shouldn't staff see the index page? seems harmless to allow that...